src/client/utils/tracking/tracking.ts

```diff
@@ -110,9 +122,20 @@ export const adtraction = (
       adt.Tag.cpn = couponCode
     }
 
-    adt.Tag.tp = isBundle(offerData)
-      ? getComboAdractionProductValue(isYouth(offerData))
-      : adtractionProductMap[offerData.quotes[0].contractType]
+    const getTpTag = () => {
+      if (isBundle(offerData))
+        return getComboAdractionProductValue(isYouth(offerData))
+      const contractType = offerData.quotes[0].contractType
+      if (!(contractType in NOT_VALID_CONTRACTS)) {
+        return adtractionProductMap[
+          contractType as TypeOfContractExcludedUnused
+        ]
+      }
+      return null
+    }
```

Comment:
Maybe we could do an early return here :) 
```suggestion
    const getTpTag = () => {
      const contractType = offerData.quotes[0].contractType
      if (contractType in NOT_VALID_CONTRACTS) {
        return null
      }

      if (isBundle(offerData)) {
        return getComboAdractionProductValue(isYouth(offerData))
      }

      return adtractionProductMap[contractType as TypeOfContractExcludedUnused]
    }
```